packages/components/src/DateTimePickers/DateRange/Inputs/Inputs.component.js

```diff
@@ -0,0 +1,88 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import omit from 'lodash/omit';
+import DebounceInput from 'react-debounce-input';
+
+import { DateRangeContext } from '../Context';
+import { START_DATE, END_DATE } from '../constants';
+import InputSizer from '../../shared/InputSizer';
+
+import Icon from '../../../Icon';
+
+import theme from './Inputs.scss';
+
+const OMIT_PROPS_INPUT = ['startInputRef', 'endInputRef'];
+
+function Inputs(props) {
+	const { startDate, endDate, inputManagement } = useContext(DateRangeContext);
```

Comment:
maybe we need to manage the case of the user doesn't define inputManagement correctly and display a specific error in the console to explain what is missing instead something like `can not read property placeholder of undefined`  ?

Reply:
and use deconstruction for inputManagement instead of use inputManagement.something

Reply:
Changed to deconstruction of `inputManagement`.
And `DateRange.Inputs` is not an individual component, it is supposed to always use under DateRangeContext. `DateRange.Manager` will provide the context value(changes comes in next PR), so it guarantee that inputManagement is passed. 😸

Reply:
not totally agree with that 😕. let's see what @yonmey or @jsomsanith will said about that 😃

Reply:
You must not expose this component in the index, so it's considered as internal, and that's sufficient for me.